experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1820,6 +1821,36 @@ describe("SharedTree", () => {
 		});
 	});
 
+	describe("Stashed ops", () => {
+		it("can apply and resubmit stashed schema ops", async () => {
+			const provider = await TestTreeProvider.create(2);
+
+			const pausedContainer: IContainerExperimental = provider.containers[0];
+			const url = (await pausedContainer.getAbsoluteUrl("")) ?? fail("didn't get url");
+			const pausedTree = provider.trees[0];
+			await provider.opProcessingController.pauseProcessing(pausedContainer);
+			pausedTree.storedSchema.update(testSchema);
+			const pendingOps = await pausedContainer.closeAndGetPendingLocalState?.();
+			provider.opProcessingController.resumeProcessing();
+
+			const loader = provider.makeTestLoader();
+			const loadedContainer = await loader.resolve({ url }, pendingOps);
+			const dataStore = await requestFluidObject<ITestFluidObject>(loadedContainer, "/");
+			const tree = await dataStore.getSharedObject<ISharedTree>("TestSharedTree");
+			await waitForContainerConnection(loadedContainer, true);
+			await provider.ensureSynchronized();
+
+			const otherLoadedTree = provider.trees[1];
+			expectSchemaEquality(tree.storedSchema, testSchema);
+			expectSchemaEquality(otherLoadedTree.storedSchema, testSchema);
+		});
+
+		function expectSchemaEquality(actual: SchemaData, expected: SchemaData): void {
```

Comment:
schematize has its own less good implementation of this: might be handy to move this to test/util so we can reuse it since this is the safer option.